writer/table.go

```diff
@@ -44,7 +44,7 @@ func (t TableWriter) Write(writer io.Writer) error {
 	table.Render()
 
 	// Disable the Output Summary if there are no outputs to display
-	if len(t.outputChanges["add"]) > 0 || len(t.outputChanges["delete"]) > 0 || len(t.outputChanges["update"]) > 0 {
```

Comment:
This logic was a bit brittle and redundant. Now it's moved to a `hasOutputChanges` helper function, which is agnostic to whether the changes are `add`, `delete`, or `update` changes; it simply returns true or false if there are _any_ non-empty `map[string][]string` output changes.

Reply:
I am not very sure if there can be any value other than add/delete/update.